hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieClusteringConfig.java

```diff
@@ -211,6 +211,12 @@ public class HoodieClusteringConfig extends HoodieConfig {
       .sinceVersion("0.9.0")
       .withDocumentation("When rewriting data, preserves existing hoodie_commit_time");
 
+  public static final ConfigProperty<Boolean> GENERATE_FILE_GROUP_ADVANCE = ConfigProperty
```

Comment:
Could you clarify why you need to assign a particular file group to a clustering group?  This defeats the purpose of clustering.